go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/vmware/terraform-provider-wavefront
 require (
 	github.com/WavefrontHQ/go-wavefront-management-api v1.14.0
 	github.com/google/go-cmp v0.5.2
-	github.com/hashicorp/terraform-plugin-sdk v1.11.0
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.4.3
 )
 
 go 1.13
```

Comment:
In a separate PR, you might want to update to go 1.6.x and make a similar change to the [release workflow](https://github.com/vmware/terraform-provider-wavefront/blob/master/.github/workflows/release.yml#L20) to enable building providers for the M1 macs (arm64).

Reply:
Will do this in a separate PR.